src/Bicep.Core/Emit/ForSyntaxValidatorVisitor.cs

```diff
@@ -90,8 +111,9 @@ public override void VisitForSyntax(ForSyntax syntax)
                 case null:
                     // this is a property loop
                     this.propertyLoopCount += 1;
+                    this.maximumPropertyLoopCount = Math.Max(this.maximumPropertyLoopCount, this.propertyLoopCount);
```

Comment:
What's the purpose of `maximumPropertyLoopCount`? It is assigned but its value doesn't seem to be used anywhere.

Reply:
Was originally trying to implement this by partially running the dane visitor logic. This was needed for that but I forgot to remove it when I found a simple way. 

I'll remove it.

Reply:
Removed it